src/libstore/store-api.hh

```diff
@@ -794,6 +794,13 @@ void copyClosure(
     CheckSigsFlag checkSigs = CheckSigs,
     SubstituteFlag substitute = NoSubstitute);
 
+void copyClosure(
```

Comment:
No doc because documented above. Just like `copyClosure` right above that where only the first overload is documented.